### PR TITLE
add searchable fields for claims

### PR DESCRIPTION
**Pull request by jvmsangkal**

### CODE REVIEW

**Comment by ravenjohn** [comment]
<a href="http://4380e460.ngrok.com/anyTV/music-dashboard/pull/379" target="_blank">ReviewNinja

File: app/scripts/translations/en-GB.js [hunk 1]
```diff
@@ -239,5 +240,8 @@ module.exports = {
 	"acknowledge_label": "Do You acknowledge that we will contact you via e-mail if you are considered for the BEAT label?",
 	"acknowledge": "I acknowledge",
 	"terms_and_conditions": "Terms & Conditions",
-	"agree": "I have read and agree to the Terms & Conditions"
+	"agree": "I have read and agree to the Terms & Conditions",
```
Comment (sibayanjasper) [inline app/scripts/translations/en-GB.js]: I think it should be `I have read and agreed to the Terms & Conditions`

Comment (ravenjohn) [inline app/scripts/translations/en-GB.js]: yep ^ please !fix

Comment (jvmsangkal) [inline app/scripts/translations/en-GB.js]: !fixed

**Comment by rycastaneda** [comment]
:star: :AllenApproves:

**Comment by ravenjohn** [comment]
:AllenApproves:

**Comment by sibayanjasper** [comment]
:AllenApproves:

**Pull request merged**